jbpm-bpmn2/src/main/java/org/jbpm/bpmn2/xml/BoundaryEventHandler.java

```diff
@@ -59,7 +59,8 @@ public Object end(final String uri, final String localName,
         Node node = (Node) parser.getCurrent();
         String attachedTo = element.getAttribute("attachedToRef");
         Attr cancelActivityAttr = element.getAttributeNode("cancelActivity");
-        boolean cancelActivity = Boolean.parseBoolean(cancelActivityAttr.getValue());
+        boolean cancelActivity = cancelActivityAttr == null ? true
+                : Boolean.parseBoolean(cancelActivityAttr.getValue());
```

Comment:
To avoid SonarCloud smell, it could be
```suggestion
        boolean cancelActivity = cancelActivityAttr == null || Boolean.parseBoolean(cancelActivityAttr.getValue());
```